### ISSUE TITLE
loom and spool executables panic

### ISSUE THREAD

**Issue by garious** [loomprotocol/loom#35]
```
 ✗ RUST_BACKTRACE=1 target/debug/loom
thread 'main' panicked at 'read: IO(Error { repr: Os { code: 35, message: "Resource temporarily unavailable" } })', src/libcore/result.rs:916:5
stack backtrace:
   0: std::sys::unix::backtrace::tracing::imp::unwind_backtrace
             at src/libstd/sys/unix/backtrace/tracing/gcc_s.rs:49
   1: std::panicking::default_hook::{{closure}}
             at src/libstd/sys_common/backtrace.rs:68
             at src/libstd/sys_common/backtrace.rs:57
             at src/libstd/panicking.rs:381
   2: std::panicking::default_hook
             at src/libstd/panicking.rs:397
   3: std::panicking::begin_panic
             at src/libstd/panicking.rs:577
   4: std::panicking::begin_panic
             at src/libstd/panicking.rs:538
   5: std::panicking::try::do_call
             at src/libstd/panicking.rs:522
   6: std::panicking::try::do_call
             at src/libstd/panicking.rs:498
   7: <core::ops::range::Range<Idx> as core::fmt::Debug>::fmt
             at src/libcore/panicking.rs:71
   8: core::result::unwrap_failed
             at /Users/travis/build/rust-lang/rust/src/libcore/macros.rs:23
   9: <core::result::Result<T, E>>::expect
             at /Users/travis/build/rust-lang/rust/src/libcore/result.rs:809
  10: loom::main
             at src/bin/loom.rs:15
```

**Comment by aeyakovenko** [comment]
yea, they don't do much yet

**Comment by aeyakovenko** [comment]
fixed in 8f8bfc6c637e78092348882a960d4f2f3d9d2b55
